src/browser/provider/built-in/base.js

```diff
@@ -9,9 +9,15 @@ function getDataFromBrowser () {
 }
 /*eslint-disable no-undef*/
 
-const INIT_SCRIPT = getDataFromBrowser.toString();
+const INIT_SCRIPT           = getDataFromBrowser.toString();
+const BROWSER_OPENING_DELAY = 3000;
```

Comment:
Please note, that our client-side scripts are executed even before the browser window is opened, and it means that most part of this timeout the browser window will be just invisible. 

Also, window opening animations can take different amount of time depending on the machine performance. On my relatively fast machine it takes from 0.5 to 1 sec to play opening animation, and on a slower machine this time may be worse. 

I think, it's possible to decrease timeout to 2 secs, but lesser values may cause instability.